crates/node/tests/it/tip20.rs

```diff
@@ -676,95 +656,3 @@ async fn test_tip20_whitelist() -> eyre::Result<()> {
 
     Ok(())
 }
```

Comment:
We should keep the `test_tip20_rewards` test, but update it to cover immediately distributed rewards now that scheduled rewards have been removed.